_sass/base.scss

```diff
@@ -39,6 +39,14 @@ a {
   @include media-query("medium") {
     padding: $spacing 0 (3 * $spacing);
   }
+
+  ul {
```

Comment:
Moving this has messed some things around when I run this branch locally. Particularly on the home page.